### ISSUE TITLE
Error TS2345

### ISSUE THREAD

**Issue by Nandhini-Saravanan** [Microsoft/TypeScript#14737]
@Component
({
selector:'my-app', 
template:`<h1>Hi there <\h1>
                   <my-tut><\my-tut>`, 
directives :[TutorialComponent] 
})

It shows the error:
Argument of type selector:string template:string; directives :typeof TutorialComponent [] ;} ' is not assignable to parameter of type Component. Object literal may only specify known properties,  and directives does not exist in type Component.

**Comment by RyanCavanaugh** [comment]
This is a problem with the Angular2 .d.ts assuming it's a problem. However I cannot find any documentation or example of using that property with the Component decorator -- are you sure it exists?

**Comment by Nandhini-Saravanan** [comment]
I saw a video on Youtube about Angular 2. It was a basic tutorial. But there he worked with Visual Studio and i used Webstorm. I just got the answer a few minutes back. There would be a file called app.module.ts in the app folder. Go there and add the Component name in the declarations column as such:

app.module.ts
@NgModule
({
//some import statement
declarations:[TutorialComponent];
//some other bootstrap component
})

The error will go. However I removed directives from the file. Maybe Component doesn't contain directives in the newer versions because only old versions of Visual Studio supports it.

Hope this helps :)
